internal/upload/filecopy/filecopy.go

```diff
@@ -73,13 +72,12 @@ func (c *Client) Upload(src string, dst string) (string, error) {
 
 		err = fs.Mkdir(destDir)
 		if err != nil {
-			return "", errors.Wrapf(err, "creating directory '%s' failed", destDir)
+			return "", fmt.Errorf("creating directory '%s' failed: %w", destDir, err)
 		}
-
 		c.debugLogFn("filecopy: created directory '%s'", destDir)
 	} else {
 		if !isDir {
-			return "", errors.Wrapf(err, "%s is not a directory", destDir)
+			return "", fmt.Errorf("%s is not a directory :%w", destDir, err)
```

Comment:
err is nil here, that was buggy before, we do not need to wrap nil
```suggestion
			return "", fmt.Errorf("%s is not a directory", destDir,)
```